tests/test_continental.py

```diff
@@ -0,0 +1,21 @@
+import pytest
+from click.testing import CliRunner
+from coastlines.continental import continental_cli
+
+def test_generate_rasters_cli():
+    runner = CliRunner()
+    result = runner.invoke(
+        continental_cli,
+        [
+            "--vector_version",
+            "{{inputs.parameters.result-version}}",
```

Comment:
@robbibt , i need help with those parameters? those are just placeholders?

Reply:
The  `--*_version` params give the outputs a unique name that is used to find and load the correct files in the final "merge" step. 

 You can set all of these options to the same string, e.g. "test":
 
 https://github.com/GeoscienceAustralia/dea-coastlines/blob/integration-test/tests/test_raster_gen.py#L14-L15
 https://github.com/GeoscienceAustralia/dea-coastlines/blob/integration-test/tests/test_vector.py#L14-L15
 https://github.com/GeoscienceAustralia/dea-coastlines/blob/integration-test/tests/test_continental.py#L10-L11

Reply:
(in operational use, we give them something logical like "v1.0.0" that represents the version number)